build.sbt

```diff
@@ -1478,6 +1478,32 @@ lazy val `polyglot-api` = project
   .dependsOn(`logging-utils`)
   .dependsOn(testkit % Test)
 
+lazy val `polyglot-api-macros` = project
```

Comment:
What is the purpose of `polyglot-api-macros` project? Do I see correctly that it contains only single source `SerdeConfig.scala`? Shouldn't it be in `polyglot-api-serde`?

Reply:
Yes, this is necessary due to `jsoniter-scala`'s usage of Scala macros. Macros have to be compiled separately from the usage site. Alternatively we could put everything into one module but then **we** would have to define some (simple) macros. I'd prefer to avoid that due to maintenance cost on our side.